components/src/themed/ThemedAvatar.tsx

```diff
@@ -8,7 +8,7 @@ const ThemedIconAvatar: typeof Avatar.Icon = (props) => {
     const style = Object.assign(
         theme.dark
             ? {
-                  backgroundColor: theme.colors.actionPalette.hover, // TODO: don't love using this because it's not a hover state
+                  backgroundColor: theme.colors.overrides.avatar?.background,
```

Comment:
Do we have a fallback value for these? What is used if the override value is undefined / how do they look?